pysteps/io/exporters.py

```diff
@@ -635,7 +645,8 @@ def initialize_forecast_exporter_netcdf(
     exporter["timestep"] = timestep
     exporter["metadata"] = metadata
     exporter["incremental"] = incremental
-    exporter["num_timesteps"] = n_timesteps
+    exporter["num_timesteps"] = num_timesteps
+    exporter["timesteps"] = n_timesteps
```

Comment:
don't we need to build the list of timesteps in case `n_timesteps` is not a list?

Reply:
It does a check later on whether this variable contains a list or an integer, so that should be covered.